src/core/server/http_resources/http_resources_service.test.ts

```diff
@@ -179,7 +177,6 @@ describe('HttpResources service', () => {
                 body: htmlBody,
                 headers: {
                   'content-type': 'text/html5',
-                  'content-security-policy': "script-src 'unsafe-eval'",
                   'x-kibana': '42',
```

Comment:
Same as other comment above

```suggestion
                  'content-security-policy': "script-src 'unsafe-eval'",
                  'x-kibana': '42',
```